src/fmu/dataio/dataio.py

```diff
@@ -560,31 +564,31 @@ class ExportData:
     # input keys (alphabetic)
     access_ssdl: dict = field(default_factory=dict)
     aggregation: bool = False
-    casepath: Union[str, Path, None] = None
+    casepath: str | Path | None = None
     config: dict = field(default_factory=dict)
-    content: Union[dict, str, None] = None
+    content: dict | str | None = None
     depth_reference: str = "msl"
-    description: Union[str, list] = ""
-    display_name: Optional[str] = None
+    description: str | list = ""
+    display_name: str | None = None
     fmu_context: str = "realization"
     forcefolder: str = ""
-    grid_model: Optional[str] = None
+    grid_model: str | None = None
     is_observation: bool = False
     is_prediction: bool = True
     name: str = ""
     undef_is_zero: bool = False
     parent: str = ""
     realization: int = -999
-    reuse_metadata_rule: Optional[str] = None
-    runpath: Union[str, Path, None] = None
+    reuse_metadata_rule: str | None = None
+    runpath: str | Path | None = None
     subfolder: str = ""
     tagname: str = ""
-    timedata: Optional[List[list]] = None
+    timedata: List[List] | None = None
```

Comment:
These must be `typing.List` due to how `_validate_variable(...)` is built. Can be converted to `list` when that functions is cleaned up.